crates/extensions/tedge_health_ext/src/lib.rs

```diff
@@ -0,0 +1,69 @@
+mod actor;
+use actor::HealthMonitorActor;
+use tedge_actors::Builder;
+use tedge_actors::DynSender;
+use tedge_actors::LinkError;
+use tedge_actors::MessageSink;
+use tedge_actors::RuntimeRequest;
+use tedge_actors::RuntimeRequestSink;
+use tedge_actors::ServiceConsumer;
+use tedge_actors::SimpleMessageBox;
+use tedge_actors::SimpleMessageBoxBuilder;
+use tedge_mqtt_ext::MqttMessage;
+use tedge_mqtt_ext::TopicFilter;
+
+type HealthMonitorMessageBox = SimpleMessageBox<MqttMessage, MqttMessage>;
+
+pub struct HealthMonitorBuilder {
+    service_name: String,
+    subscriptions: TopicFilter,
+    box_builder: SimpleMessageBoxBuilder<MqttMessage, MqttMessage>,
+}
+
+impl HealthMonitorBuilder {
```

Comment:
In order to ease writing of a `main.rs` with health checks, one can add a method to this builder to update an MQTT config with init and last will messages.

```
   pub fn set_init_and_last_will(&self, config: MqttConfig) -> MqttConfig {
      config
          .with_initial_message(|| health_status_up_message(self.service_name))
         .with_last_will_message(health_status_down_message(self.service_name))
   }
```

Reply:
But the `HealthMonitorBuilder` has to be created first, but this depends on `mqtt_actor`. This leads to a cyclic dependency.

Reply:
Added